pkg/ebpf/events_processor.go

```diff
@@ -283,7 +274,12 @@ func (t *Tracee) processCgroupMkdir(event *trace.Event) error {
 		// If cgroupId is from a regular cgroup directory, and not the
 		// container base directory (from known runtimes), it should be
 		// removed from the containers bpf map.
-		if err := t.containers.RemoveFromBpfMap(t.bpfModule, cgroupId, hId); err != nil {
+		err := capabilities.GetInstance().EBPF(
```

Comment:
I assume this can be a hotpath in certain workloads. Maybe it's worth adding to the `Base` ring when tracing `cgroup_mkdir`? Although that means for most users the `EBPF` ring will be always effective.

Reply:
Yep I wanted to avoid keeping BPF rights here. If this proves to be a bottleneck (specially now that we'll have an easy pyroscope dashboard) then we can think of something else.